source/simulator/stokes_matrix_free.cc

```diff
@@ -1232,6 +1232,12 @@ namespace aspect
       prm.declare_entry ("Output details", "false",
                          Patterns::Bool(),
                          "Turns on extra information for the matrix free GMG solver to be printed.");
+      prm.declare_entry ("Execute solver timings", "false",
+                         Patterns::Bool(),
+                         "Executes different parts of the Stokes solver repeatedly and print timing information."
+                         "This is for internal benchmarking purposes, it's useful if you want to see how the solver"
+                         "performs. Otherwise, you don't want to enable this, since it adds additional computatinal costs"
+                         "to get the timing information.");
```

Comment:
```suggestion
                         "Executes different parts of the Stokes solver repeatedly and print timing information. "
                         "This is for internal benchmarking purposes: It is useful if you want to see how the solver "
                         "performs. Otherwise, you don't want to enable this, since it adds additional computational cost "
                         "to get the timing information.");
```